host-bin/startxiwi

```diff
@@ -0,0 +1,93 @@
+#!/bin/sh -e
+# Copyright (c) 2015 The crouton Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+set -e
+
+APPLICATION="${0##*/}"
+OPTS_ENTER=''
+OPTS_XIWI=''
+USAGE="$APPLICATION [options] CHROOT_APP [options]
+
+Wraps enter-chroot to launch a xiwi-app in a new windowed or tabbed session.
+You can launch any individual chroot graphical application in crouton windows.
+
+By default, it will log into the primary user on the first chroot found.
+
+Enter '-h' for 'enter-chroot' options and enter '-H' for 'xiwi-app' options.
```

Comment:
I think -h should just have two sections: enter-chroot options and xiwi options.
`enter-chroot` options should be outputted by actually running `enter-chroot -h` and selecting the relevant options.  Perhaps:

```
Chroot options:
`enter-chroot -h 2>&1 | grep -e ' -[bckntu]'`
```